tests/ui/single_component_path_imports.rs

```diff
@@ -1,5 +1,5 @@
 #![warn(clippy::single_component_path_imports)]
-#![allow(unused_imports)]
+#![allow(unused_imports, clippy::literal_string_with_formatting_args)]
```

Comment:
I believe this is no longer needed since `4` in `{4}` isn't a valid identifier

Same for other tests. Could you remove them to make sure those FPs are resolved?

Reply:
Sure, let me go through them. :+1: